scripts/ci/check_compliance.py

```diff
@@ -1084,6 +1087,48 @@ def run(self):
             except MaintainersError as ex:
                 self.failure(f"Error parsing {file}: {ex}")
 
+class ModulesMaintainers(ComplianceTest):
+    """
+    Check that all modules have a MAINTAINERS entry.
+    """
+    name = "ModulesMaintainers"
+    doc = "Check that all modules have a MAINTAINERS entry."
+    path_hint = "<git-top>"
+
+    def run(self):
+        MAINTAINERS_FILES = ["MAINTAINERS.yml", "MAINTAINERS.yaml"]
+
+        manifest = Manifest.from_file()
+
+        try:
+            manifest_file = Path(manifest.path).relative_to(ZEPHYR_BASE)
+        except ValueError:
+            return
+
+        if str(manifest_file) not in get_files(filter="d"):
```

Comment:
This is looking for the manifest file in the `git diff --name-only` output, which makes it seem like the expectation is that `manifest_file` is in the repository itself, but then why compute it relative to ZEPHYR_BASE?

I looked in the commit log, but it's not clear if this is only meant to be run for the zephyr repository, or if it's a check that downstream users can use as well to figure out if they have their own sort of maintainer file violations; what's the idea here?

Reply:
Yeah good point that's an unnecesesary restriction. On a second (third?) thought thogh I'm thinking we are better off without this check entirely and just always run it, so we'd catch any potential missing area starting from maintainers file change as well (unlikely, but who knows).